offchainreporting/gethwrappers/go_generate.go

```diff
@@ -0,0 +1,10 @@
+// Package gethwrappers keeps track of the golang wrappers of the solidity contracts
```

Comment:
this is cool!

Reply:
this has been replaced by a makefile in the offchainreporting repo, could we pull in those changes?

Reply:
Not yet